SUPPORTED_LANGUAGES.md

```diff
@@ -130,6 +130,7 @@ The table below shows the full list of languages (and corresponding classes/alia
 | LiveScript              | livescript, ls         |         |
 | LookML                  | lookml                 | [highlightjs-lookml](https://github.com/spectacles-ci/highlightjs-lookml) |
 | Lua                     | lua                    |         |
+| Luau                    | luau                   | [highlightjs-luau](https://github.com/RobloxianDemo/highlightjs-luau) |
```

Comment:
Needs to link to our org since you have an official repo here.  If you don't want one here, then happy to link to your repo.

Reply:
I'll do that now, 👍

Reply:
I updated the link [here](https://github.com/highlightjs/highlight.js/pull/3923/commits/13d24cfabe35d18e7ab925aab42fef6e4e97f862).